crates/tracing_macros/src/autotrace.rs

```diff
@@ -1,13 +1,39 @@
+extern crate newrelic;
 use proc_macro::TokenStream;
 use quote::quote;
+use std::env;
 use syn::{Attribute, ImplItemMethod, ItemFn};
 
 const DEBUG_OUTPUT: bool = false;
 
 #[derive(Default)]
-pub(crate) struct Autotrace {}
+pub(crate) struct Autotrace {
+    app_name: String,
+    license_key: Option<String>,
```

Comment:
??? Are we paying for this software?

Reply:
We are free tier at this point but I am not so sure what it will be like in the future